### PR TITLE
Update yml file in local install instructions

### PR DESCRIPTION
**Pull request by ArturoAmorQ**
As mentioned in [this forum post](https://mooc-forums.inria.fr/moocsl/t/get-feature-names-out/10331) our local install instruction led people to use `sklearn version 0.24.2`, which has some deprecated notations.

This PR fixes it.

### CODE REVIEW

File: local-install-instructions.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Using python in /home/lesteve/miniconda3/envs/scikit-learn-course
 [ OK ] numpy version 1.19.5
 [ OK ] scipy version 1.6.0
 [ OK ] matplotlib version 3.3.3
-[ OK ] sklearn version 0.24.0
+[ OK ] sklearn version 1.1.2
```
Comment (glemaitre) [inline local-install-instructions.md:49]:
```suggestion
[ OK ] sklearn version 1.0.2
```

**Comment by lesteve** [comment]
The link to the forum post seems off: it goes to https://lms.fun-mooc.fr/courses/course-v1:inria+41026+session02/6ad122825f0f4241a47f1511411e65c8/ ... can you correct it?

If we indeed require scikit-learn >= 1 for some of our notebooks, we should also update https://github.com/INRIA/scikit-learn-mooc/blob/main/check_env.py

**Comment by ArturoAmorQ** [comment]
> The link to the forum post seems off: it goes to https://lms.fun-mooc.fr/courses/course-v1:inria+41026+session02/6ad122825f0f4241a47f1511411e65c8/ ... can you correct it?
> 
> If we indeed require scikit-learn >= 1 for some of our notebooks, we should also update https://github.com/INRIA/scikit-learn-mooc/blob/main/check_env.py

You are right. It's done, thanks!

**Comment by lesteve** [comment]
You may as well do something similar in `requirements*.txt` and `environment-dev.yml`. I would use `1.0.2` everywhere for consistency.

**Comment by lesteve** [comment]
Merging, thanks!

**Pull request merged**